rust_src/src/lisp.rs

```diff
@@ -429,6 +431,7 @@ impl LispObject {
             wrong_type!(Qvectorp, self)
         }
     }
+    */
```

Comment:
A belated +1 from me: it's nice to see the code get less verbose.

Why did you comment out code though? Can we delete it?

Reply:
I expected they would get used. It can go away if that turns out to be a false idea